Proxy containers: using a custom container image for a service (#2014)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/proxy-container-installation.adoc

- [literal]``TAG``=[path]``latest``

=== Using a custom container image for a service

> By default, the {productname} Proxy suite is set to use the same image version and registry path for each of its services. However, there's a possibility to change this behavior by overriding the default values for a specific service. The [literal]``uyuni-proxy`` CLI bundled with the package, has an [literal]``update image`` command taking the following arguments:

Reviewer: ```suggestion
By default, the {productname} Proxy suite is set to use the same image version and registry path for each of its services.
However, it is possible to override the default values for a specific service.
The [literal]``uyuni-proxy`` CLI bundled with the package, has an [literal]``update image`` command  with the following parameters:
```

- [literal]``-s`` for the service name
- [literal]``-t`` for the version tag
- [literal]``-r`` for the registry path

> An example usage would look like this:

Reviewer: ```suggestion
For example, use it like this:

```

----
uyuni-proxy update image -s httpd -t 0.1.0 -r registry.opensuse.org/uyuni
----
It would adjust a configuration file for the httpd service, taking [path]``registry.opensuse.org/uyuni`` as the registry and [literal]``0.1.0`` as the version tag and then restart it.

> To reset the values to defaults, run the proxy reset command, specifying the service with the -s flag. An example usage would look like the following:

Reviewer: ```suggestion
To reset the values to defaults, run the proxy reset command, specifying the service with the [literal]``-s`` parameter.
For example, use it like this:
```

----
uyuni-proxy reset -s httpd
----

> This command would reset the configuration of the [literal]``httpd`` service to the global defaults and restarts it.

Reviewer: ```suggestion
This command would reset the configuration of the [literal]``httpd`` service to the global defaults and then restart it.
```

> See the [literal]``uyuni-proxy --help`` for more detail.

Reviewer: ```suggestion
For more information, see the [literal]``uyuni-proxy --help``.
```
